Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -535,6 +537,15 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
     [super observeValueForKeyPath:keyPath ofObject:object change:change context:context];
 }
 
+- (BOOL)isBlockedByScreenTime
+{
+#if ENABLE(SCREEN_TIME)
+    return _isBlockedByScreenTime;
+#else
+    return NO;
+#endif
+}
```

Comment:
Pretty sure this is unnecessary, since the getter should be automatically generated.